modules/build/src/main/scala/scala/build/bsp/BspServer.scala

```diff
@@ -254,6 +259,55 @@ class BspServer(
 
   override def onBuildExit(): Unit = ()
 
+  // TODO: reload dependencies
+  // TODO: reload scalac & javac options
+  // TODO: fix IntelliJ run tasks not picking up new sources
+  override def workspaceReload(): CompletableFuture[Object] =
+    super.workspaceReload().thenApply[Object] { res =>
+      val buildTargetsToRecompile: List[b.BuildTargetIdentifier] =
+        for { // this is just a workaround hack to emulate workspace reloading until Bloop itself supports it
+          case (targetUri, targetName) <- buildTargetNamesByUri.toList
+          scalaBuildDirPath = os.Path(new URI(targetUri).getRawPath)
```

Comment:
I don't think this way of converting URIs to paths will work in all cases (on Windows in particular). This should work better:
```suggestion
          scalaBuildDirPath = os.Path(Paths.get(new URI(targetUri)))
```
(with `Paths` being `java.nio.file.Paths`)